css-brochure-form/styles.css

```diff
@@ -0,0 +1,74 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+body {
+  background-color: white;
+}
+
+h3 {
+  text-align: center;
+  font-family: "Brandon Grotesque";
+  font-style: normal;
+  font-weight: 500;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
```

Comment:
These are only necessary on the `@font-face` rules I made, you don't need them to use the font family, just `font-family`